python/servo/build_commands.py

```diff
@@ -339,15 +339,15 @@ def build(self, target=None, release=False, dev=False, jobs=None,
             shutil.copy(path.join(self.android_support_dir(), "openssl.makefile"), openssl_dir)
             shutil.copy(path.join(self.android_support_dir(), "openssl.sh"), openssl_dir)
 
-            # Check if the NDK version is 12
+            # Check if the NDK version is 15
             if not os.path.isfile(path.join(env["ANDROID_NDK"], 'source.properties')):
                 print("ANDROID_NDK should have file `source.properties`.")
                 print("The environment variable ANDROID_NDK may be set at a wrong path.")
                 sys.exit(1)
             with open(path.join(env["ANDROID_NDK"], 'source.properties')) as ndk_properties:
                 lines = ndk_properties.readlines()
-                if lines[1].split(' = ')[1].split('.')[0] != '12':
-                    print("Currently only support NDK 12.")
+                if lines[1].split(' = ')[1].split('.')[0] != '15':
+                    print("Currently only support NDK 15.")
```

Comment:
Probably should also say something like "please delete the android-toolchains folder and rerun bootstrap-android"